src/App.js

```diff
@@ -27,7 +26,7 @@ function App() {
   }, []);
 
   useEffect(() => {
-    const url = `${process.env.REACT_APP_API_URL}/projects`;
+    const url = `${process.env.REACT_APP_API_URL}/users`;
```

Comment:
Nice!

On another note, do you have multiple `useEffect`'s in App.js? If you do, that's totally fine, but a cleaner approach might be to separate the fetch calls into their own functions and call them all in one useEffect. E.g.:

```
function getUsers() {
    fetch(...)
}

function getProjects() {
    fetch(...)
}

useEffect(() => {
    getUsers();
    getProjects();
}, [])
```